src/github3/api.py

```diff
@@ -17,6 +18,10 @@ def authorize(username, password, scopes, note='', note_url='', client_id='',
               client_secret='', two_factor_callback=None, github=None):
     """Obtain an authorization token for the GitHub API.
 
+    .. deprecated:: 1.2.0
+
+        Use :meth:`github3.GitHub.authorize` instead.
```

Comment:
These are documented as `github3.github.GitHub.authorize` so that should be used elsewhere (e.g., `github3.github.GitHub.<method>`

Reply:
Whoops! Will fix this.